examples/07-EMIT/interference_gui.py

```diff
@@ -566,8 +578,11 @@ def closeEvent(self, event):
         msg.setWindowTitle("Closing GUI")
         msg.setText("Closing AEDT, please wait for GUI to close on its own.")
```

Comment:
We should omit "please" from the message. This word isn't necessary in technical documentation. I'd suggest rewording the message to "Closing AEDT. Wait for the GUI to close on its own."